packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -65,13 +65,14 @@ export interface WalletProps {
     providerType: ProviderType;
     screenWidth: ScreenWidths;
     location: Location;
+    trackedTokenStateByAddress: TokenStateByAddress;
     onToggleLedgerDialog: () => void;
     onAddToken: () => void;
     onRemoveToken: () => void;
+    refetchTokenStateAsync: (tokenAddress: string) => Promise<void>;
 }
 
 interface WalletState {
-    trackedTokenStateByAddress: TokenStateByAddress;
```

Comment:
can we get rid of the `trackedTokens` prop as well now?

Reply:
We cannot as we use that to render, although I agree it would be nice to consolidate trackedTokens and trackedTokenStateByAddress.